src/main/java/athena/Task.java

```diff
@@ -0,0 +1,87 @@
+package athena;
+
+public class Task {
+    public static final String YES = "Y";
+    public static final String NO = "N";
+    private String name;
+
+    private String startTime;
+    private String duration;
+    private String deadline;
+
+    private String recurrence;
+    private boolean isDone = false;
+    private String importance;
+    private String notes;
+
+    private String getStatus() {
+        return (isDone ? YES : NO);
+    }
+
+    public Task(String name, String startTime, String duration, String deadline,
+            String recurrence, String importance, String notes) {
+        this.name = name;
+        this.startTime = startTime;
+        this.duration = duration;
+        this.deadline = deadline;
+        this.recurrence = recurrence;
+        this.importance = importance;
+        this.notes = notes;
+    }
+
+
+    /**
+     * Edits the features of the task.
+     *
+     * @param name New task name
+     * @param startTime New task start time
+     * @param duration New task duration
+     * @param deadline New task deadline
+     * @param recurrence New task recurrence
+     * @param importance New task importance
+     * @param notes New task notes
+     */
+    public void changeAttributesTo(String name, String startTime, String duration,
```

Comment:
The `To` at the end of the method name feels unnecessary. Shall we use `changeAttributes`? Or `editAttributes` or `edit` which I think will match the user functionality more closely too.

Reply:
Noted! Has been changed to `edit`